backend/scripts/aws_credentials_manager.py

```diff
@@ -0,0 +1,133 @@
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
+
+import boto3
+import logging
+import subprocess
+import time
+from typing import Dict
+from botocore.exceptions import ClientError, NoCredentialsError, PartialCredentialsError
+
+# used to log info to terminal
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+# AWS credentials manager class
+class AWSCredentialsManager:
+    """Manages AWS credentials validation and retrieval."""
+    
+    def __init__(self):
+        self.session = boto3.Session()
+        self.region = self.session.region_name or 'us-east-1'
```

Comment:
Do we have global constant? The aws region need to be consistent used for all other aws resources.

We can make 'us-east-1' as constant to use here, and add it to logs to say no user aws region provided, use default region 'us-east-1'.